libr/include/r_util/r_big.h

```diff
@@ -1,41 +1,99 @@
 #ifndef R_BIG_H
 #define R_BIG_H
 
+#include "../r_types.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 #if HAVE_LIB_GMP
+/* Use GMP's data struct */
 #define RNumBig mpz_t
-#elif HAVE_LIB_SSL
-#define RNumBig BIGNUM
 #else
-#define	R_BIG_SIZE 10000
+/* Use default impl */
```

Comment:
so much defines without consistent namings, this may conflict with other projects including r_utl